lib/bundle.js

```diff
@@ -87,7 +89,7 @@ exports.bundle = function(moduleExpression, fileName, opts) {
 
     if (opts.inject) {
       // Add the bundle to config if the inject flag was given.
-      var bundleName = extractBundleName(fileName);
+      var bundleName = extractBundleName(fileName, systemBuilder);
```

Comment:
I believe, the func `systemBuilder.getCannonicalName(...)` needs a proper `file URL` so, if we inline this we probably have to do something like:
``` javascript
 var bundleName = builder.getCanonicalName( toFileURL(path.resolve(config.pjson.baseURL, fileName).replace(/\\/g, '/'))); 
``
This is mostly what `extractBundleName` is doing now.

Please let me know, if you prefer the above approach. I will make the change.